web/src/widgets/pokemon.tsx

```diff
@@ -176,3 +177,5 @@ const Tile = ({ children, color }: { children: React.ReactNode; color: string })
 };
 
 export default App;
+
+mountWidget(<App />);
```

Comment:
pour rester cohérent ?

```suggestion
mountWidget(<StrictMode>
    <Pokemon />
  </StrictMode>,);
```